### PR TITLE
add draft diagrams

### PR DESCRIPTION
**Pull request by ellenbechtel**
Changes made: added draft diagrams
-----------
Specifically, I added sketches of a SWE explainer (which hopefully will be in D3 one day) and four panels of a conceptual diagram.

Testing:
----------------------------
Before making this pull request, I:
- [x] Cleaned the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Made sure all tests run
- [ ] Ran WAVE plugin 508 compliance tool

I can confirm this has been checked on:
- [x] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,17993 +1,8 @@
 {
   "name": "snow-to-flow",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (collnell) [inline package-lock.json:4]: Is this intentional? Curious about what it means

**Comment by collnell** [comment]
There are lots of changes in the `package-lock.json` file that I'm guessing are related to the lint fix or dropping dependencies? Otherwise, looks good to me. Moving things along!

**Comment by collnell** [comment]
This PR addresses https://github.com/USGS-VIZLAB/snow-to-flow/issues/20

File: package-lock.json (hunk 1 above)
Comment (ellenbechtel) [reply package-lock.json:4]: ...hmm. no idea

**Comment by ellenbechtel** [comment]
This PR didn't run "npm run build" for Ellen's local computer, but after some back and forth we decided to merge it anyway because we can always rebuild package-lock.json

**Pull request merged**